### PR TITLE
Update oVirt requirement to v4.1

### PR DESCRIPTION
**Pull request by tremble**
The Ansible ovirt_disk module, shipped as part of Ansible 2.3, as used by this role sends 'direction' when
initiating the transfer, this is not supported until 4.1

### CODE REVIEW

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for an oVirt project! 

I am the bot who triggers CI builds on the oVirt Jenkins server. 
In order to allow this PR to run fully automated tests, you can do one of the following options:

1. Asking one of the project maintainers to add you to the project team ( it's done via GitHub admin page for the project )
2. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the oVirt organization, so you'll be able to run tests for all oVirt's projects.
3. Asking one of the maintainers to run the 'ci test please' on the PR.


Note: You might see multiple messages like this showing up on a single PR, this is due to a known bug in the GitHub trigger plugin on Jenkins, the oVirt infra team is working on fixing this via new testing flow in the near future, so we'll be able to post a single message per PR.

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for an oVirt project! 

I am the bot who triggers CI builds on the oVirt Jenkins server. 
In order to allow this PR to run fully automated tests, you can do one of the following options:

1. Asking one of the project maintainers to add you to the project team ( it's done via GitHub admin page for the project )
2. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the oVirt organization, so you'll be able to run tests for all oVirt's projects.
3. Asking one of the maintainers to run the 'ci test please' on the PR.


Note: You might see multiple messages like this showing up on a single PR, this is due to a known bug in the GitHub trigger plugin on Jenkins, the oVirt infra team is working on fixing this via new testing flow in the near future, so we'll be able to post a single message per PR.

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for an oVirt project! 

I am the bot who triggers CI builds on the oVirt Jenkins server. 
In order to allow this PR to run fully automated tests, you can do one of the following options:

1. Asking one of the project maintainers to add you to the project team ( it's done via GitHub admin page for the project )
2. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the oVirt organization, so you'll be able to run tests for all oVirt's projects.
3. Asking one of the maintainers to run the 'ci test please' on the PR.


Note: You might see multiple messages like this showing up on a single PR, this is due to a known bug in the GitHub trigger plugin on Jenkins, the oVirt infra team is working on fixing this via new testing flow in the near future, so we'll be able to post a single message per PR.

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for an oVirt project! 

I am the bot who triggers CI builds on the oVirt Jenkins server. 
In order to allow this PR to run fully automated tests, you can do one of the following options:

1. Asking one of the project maintainers to add you to the project team ( it's done via GitHub admin page for the project )
2. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the oVirt organization, so you'll be able to run tests for all oVirt's projects.
3. Asking one of the maintainers to run the 'ci test please' on the PR.


Note: You might see multiple messages like this showing up on a single PR, this is due to a known bug in the GitHub trigger plugin on Jenkins, the oVirt infra team is working on fixing this via new testing flow in the near future, so we'll be able to post a single message per PR.

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for an oVirt project! 

I am the bot who triggers CI builds on the oVirt Jenkins server. 
In order to allow this PR to run fully automated tests, you can do one of the following options:

1. Asking one of the project maintainers to add you to the project team ( it's done via GitHub admin page for the project )
2. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the oVirt organization, so you'll be able to run tests for all oVirt's projects.
3. Asking one of the maintainers to run the 'ci test please' on the PR.


Note: You might see multiple messages like this showing up on a single PR, this is due to a known bug in the GitHub trigger plugin on Jenkins, the oVirt infra team is working on fixing this via new testing flow in the near future, so we'll be able to post a single message per PR.

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for an oVirt project! 

I am the bot who triggers CI builds on the oVirt Jenkins server. 
In order to allow this PR to run fully automated tests, you can do one of the following options:

1. Asking one of the project maintainers to add you to the project team ( it's done via GitHub admin page for the project )
2. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the oVirt organization, so you'll be able to run tests for all oVirt's projects.
3. Asking one of the maintainers to run the 'ci test please' on the PR.


Note: You might see multiple messages like this showing up on a single PR, this is due to a known bug in the GitHub trigger plugin on Jenkins, the oVirt infra team is working on fixing this via new testing flow in the near future, so we'll be able to post a single message per PR.

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for an oVirt project! 

I am the bot who triggers CI builds on the oVirt Jenkins server. 
In order to allow this PR to run fully automated tests, you can do one of the following options:

1. Asking one of the project maintainers to add you to the project team ( it's done via GitHub admin page for the project )
2. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the oVirt organization, so you'll be able to run tests for all oVirt's projects.
3. Asking one of the maintainers to run the 'ci test please' on the PR.


Note: You might see multiple messages like this showing up on a single PR, this is due to a known bug in the GitHub trigger plugin on Jenkins, the oVirt infra team is working on fixing this via new testing flow in the near future, so we'll be able to post a single message per PR.

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for an oVirt project! 

I am the bot who triggers CI builds on the oVirt Jenkins server. 
In order to allow this PR to run fully automated tests, you can do one of the following options:

1. Asking one of the project maintainers to add you to the project team ( it's done via GitHub admin page for the project )
2. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the oVirt organization, so you'll be able to run tests for all oVirt's projects.
3. Asking one of the maintainers to run the 'ci test please' on the PR.


Note: You might see multiple messages like this showing up on a single PR, this is due to a known bug in the GitHub trigger plugin on Jenkins, the oVirt infra team is working on fixing this via new testing flow in the near future, so we'll be able to post a single message per PR.

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for an oVirt project! 

I am the bot who triggers CI builds on the oVirt Jenkins server. 
In order to allow this PR to run fully automated tests, you can do one of the following options:

1. Asking one of the project maintainers to add you to the project team ( it's done via GitHub admin page for the project )
2. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the oVirt organization, so you'll be able to run tests for all oVirt's projects.
3. Asking one of the maintainers to run the 'ci test please' on the PR.


Note: You might see multiple messages like this showing up on a single PR, this is due to a known bug in the GitHub trigger plugin on Jenkins, the oVirt infra team is working on fixing this via new testing flow in the near future, so we'll be able to post a single message per PR.

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for an oVirt project! 

I am the bot who triggers CI builds on the oVirt Jenkins server. 
In order to allow this PR to run fully automated tests, you can do one of the following options:

1. Asking one of the project maintainers to add you to the project team ( it's done via GitHub admin page for the project )
2. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the oVirt organization, so you'll be able to run tests for all oVirt's projects.
3. Asking one of the maintainers to run the 'ci test please' on the PR.


Note: You might see multiple messages like this showing up on a single PR, this is due to a known bug in the GitHub trigger plugin on Jenkins, the oVirt infra team is working on fixing this via new testing flow in the near future, so we'll be able to post a single message per PR.

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for an oVirt project! 

I am the bot who triggers CI builds on the oVirt Jenkins server. 
In order to allow this PR to run fully automated tests, you can do one of the following options:

1. Asking one of the project maintainers to add you to the project team ( it's done via GitHub admin page for the project )
2. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the oVirt organization, so you'll be able to run tests for all oVirt's projects.
3. Asking one of the maintainers to run the 'ci test please' on the PR.


Note: You might see multiple messages like this showing up on a single PR, this is due to a known bug in the GitHub trigger plugin on Jenkins, the oVirt infra team is working on fixing this via new testing flow in the near future, so we'll be able to post a single message per PR.

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for an oVirt project! 

I am the bot who triggers CI builds on the oVirt Jenkins server. 
In order to allow this PR to run fully automated tests, you can do one of the following options:

1. Asking one of the project maintainers to add you to the project team ( it's done via GitHub admin page for the project )
2. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the oVirt organization, so you'll be able to run tests for all oVirt's projects.
3. Asking one of the maintainers to run the 'ci test please' on the PR.


Note: You might see multiple messages like this showing up on a single PR, this is due to a known bug in the GitHub trigger plugin on Jenkins, the oVirt infra team is working on fixing this via new testing flow in the near future, so we'll be able to post a single message per PR.

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for an oVirt project! 

I am the bot who triggers CI builds on the oVirt Jenkins server. 
In order to allow this PR to run fully automated tests, you can do one of the following options:

1. Asking one of the project maintainers to add you to the project team ( it's done via GitHub admin page for the project )
2. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the oVirt organization, so you'll be able to run tests for all oVirt's projects.
3. Asking one of the maintainers to run the 'ci test please' on the PR.


Note: You might see multiple messages like this showing up on a single PR, this is due to a known bug in the GitHub trigger plugin on Jenkins, the oVirt infra team is working on fixing this via new testing flow in the near future, so we'll be able to post a single message per PR.

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for an oVirt project! 

I am the bot who triggers CI builds on the oVirt Jenkins server. 
In order to allow this PR to run fully automated tests, you can do one of the following options:

1. Asking one of the project maintainers to add you to the project team ( it's done via GitHub admin page for the project )
2. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the oVirt organization, so you'll be able to run tests for all oVirt's projects.
3. Asking one of the maintainers to run the 'ci test please' on the PR.


Note: You might see multiple messages like this showing up on a single PR, this is due to a known bug in the GitHub trigger plugin on Jenkins, the oVirt infra team is working on fixing this via new testing flow in the near future, so we'll be able to post a single message per PR.

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for an oVirt project! 

I am the bot who triggers CI builds on the oVirt Jenkins server. 
In order to allow this PR to run fully automated tests, you can do one of the following options:

1. Asking one of the project maintainers to add you to the project team ( it's done via GitHub admin page for the project )
2. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the oVirt organization, so you'll be able to run tests for all oVirt's projects.
3. Asking one of the maintainers to run the 'ci test please' on the PR.


Note: You might see multiple messages like this showing up on a single PR, this is due to a known bug in the GitHub trigger plugin on Jenkins, the oVirt infra team is working on fixing this via new testing flow in the near future, so we'll be able to post a single message per PR.

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for an oVirt project! 

I am the bot who triggers CI builds on the oVirt Jenkins server. 
In order to allow this PR to run fully automated tests, you can do one of the following options:

1. Asking one of the project maintainers to add you to the project team ( it's done via GitHub admin page for the project )
2. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the oVirt organization, so you'll be able to run tests for all oVirt's projects.
3. Asking one of the maintainers to run the 'ci test please' on the PR.


Note: You might see multiple messages like this showing up on a single PR, this is due to a known bug in the GitHub trigger plugin on Jenkins, the oVirt infra team is working on fixing this via new testing flow in the near future, so we'll be able to post a single message per PR.

**Comment by tremble** [comment]
Requirement more specifically introduced by https://github.com/ansible/ansible/pull/22101

File: roles/ovirt-image-template/README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Requirements
 
  * Ansible version 2.3 or higher.
  * Python SDK version 4 or higher.
```
Comment (machacekondra) [inline roles/ovirt-image-template/README.md:10]: And so Python SDK must be 4.1 or higher.

**Review by machacekondra — COMMENTED** [review]
Thanks for patch.

**Comment by machacekondra** [comment]
ci build please

**Comment by machacekondra** [comment]
Merged.

**Pull request closed without merging**